index.html

```diff
@@ -132,7 +132,7 @@ <h1>steam.tools</h1>
 	<footer id="copyright">
 		<div class="rightbar" style="display: inline-block; text-align: right;">
 			Website by <a target="_blank" href="https://ehsankia.com">Ehsan Kia</a><br>
-			<a target="_blank" href="https://steam.tools">steam.tools</a> &copy; 2013-2018
+			<a target="_blank" href="https://steam.tools">steam.tools</a> &copy; 2013-2020

```

Comment:
<span id="year">2020</span>

<script>
    document.getElementById("year").innerHTML = new Date().getFullYear();
</script>

Reply:
how am i supposed to get my yearly 4 PRs in october if it's automated!

Reply:
Do you have your 4 pulls? You can send me one for this? 😀